Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -1122,6 +1127,32 @@ void CGameListCtrl::OnWiki(wxCommandEvent& WXUNUSED (event))
 	std::string wikiUrl = "https://wiki.dolphin-emu.org/dolphin-redirect.php?gameid=" + iso->GetUniqueID();
 	WxUtils::Launch(wikiUrl);
 }
+void CGameListCtrl::OnHostNetplay(wxCommandEvent& WXUNUSED (event))
+{
+	const GameListItem* iso = GetSelectedISO();
+	if (!iso)
+		return;
+	
+	NetPlaySetupDiag* npsd = new NetPlaySetupDiag(this, this);
+	// Lang needs to be consistent
+	auto const lang = 0;
+	
+	std::string name(iso->GetName(lang));
+	
+	if (iso->GetRevision() != 0)
+		name = name + " (" + iso->GetUniqueID() + ", Revision " + std::to_string((long long)iso->GetRevision()) + ")";
+	else
+		name = name + " (" + iso->GetUniqueID() + ")";
+	
+	IniFile inifile;
+	inifile.Load(File::GetUserPath(D_CONFIG_IDX) + "Dolphin.ini");
+	IniFile::Section& netplay_section = *inifile.GetOrCreateSection("NetPlay");
+	
+	std::string port;
+	netplay_section.Get("HostPort", &port, "2626");
+	npsd->HostGame(name, stol(port));
```

Comment:
You should introduce this configuration variable into the `ConfigManager` instead of reading the ini file yourself here.